src/views/assets/Asset/AssetDetail/PermUser.vue

```diff
@@ -16,7 +16,7 @@
 </template>
 
 <script>
-import ListTable from '@/components/Table/ListTable'
+import { DrawerListTable as ListTable } from '@/components'
 import { GenericListTableDialog } from '@/layout/components'
 import { DetailFormatter } from '@/components/Table/TableFormatters'
 import PermUserGroupCard from './components/PermUserGroupCard'
```

Comment:
The code appears to be quite complex due to importing various components and modules within an environment context, which might not allow for simple differences or optimizations in this instance. Please provide more information about the specific sections of the code that you believe need examination if they contain such elements or patterns.